packages/date-picker/src/utils/validateDate.ts

```diff
@@ -53,12 +53,13 @@ export const validateDate = (
       validationResponse: {
         ...baseResponse,
         status: "error",
-        validationMessage: `${newInputValue} is not available, try another date`,
+        validationMessage: `${inputValue} is not available, try another date`,
```

Comment:
Should this also check if `inputValue` is undefined?

Reply:
Nope because a valid disabled date is not empty :)

Reply:
Ah right, didn't check the rest of the file 😅